### PR TITLE
Add LLVM_LIBDIR_SUFFIX for /usr/lib64

### PR DESCRIPTION
**Pull request by xiangzhai**
Hi @kiranchandramohan

Fix #193

Please review it.

Thanks,
Leslie Zhai

### CODE REVIEW

File: runtime/flangrti/CMakeLists.txt [hunk 1]
```diff
@@ -91,6 +91,9 @@ if (NOT DEFINED LIBOMP_EXPORT_DIR)
     libomp.so
     HINTS ${CMAKE_BINARY_DIR}/lib)
   target_link_libraries(flangrti_shared ${FLANG_LIBOMP})
+else()
+  link_directories(${LIBOMP_EXPORT_DIR})
+  target_link_libraries(flangrti_shared omp)
```
Comment (kiranchandramohan) [inline runtime/flangrti/CMakeLists.txt:96]: Could you explain this change?

Comment (xiangzhai) [reply runtime/flangrti/CMakeLists.txt:96]: Add openmp library link custom directories.

Is it better to split the openmp part to another patch?

Thanks,
Leslie Zhai

Comment (kiranchandramohan) [reply runtime/flangrti/CMakeLists.txt:96]: Yes, if it is addressing a different issue than a separate patch is better.

BTW, may i ask what is your interest in Flang? Is it hobby or professional work?

Comment (xiangzhai) [reply runtime/flangrti/CMakeLists.txt:96]: Hobby :)

I [migrated dragonegg](https://reviews.llvm.org/D35667) .  And [Chad Rosier](https://reviews.llvm.org/p/mcrosier/) introduced flang.

Thanks,
Leslie Zhai

**Pull request closed without merging**